plot_gamma_poisson_reg.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
This should go in the ``examples`` folder, right?

Reply:
yes, it's WIP